35/generated/mirror_source_config.html

```diff
@@ -440,6 +440,16 @@ <h4><a id="replication.policy.class"></a><a id="mirror_source_replication.policy
 </tbody></table>
 </li>
 <li>
+<h4><a id="replication.policy.internal.topic.separator.enabled"></a><a id="mirror_source_replication.policy.internal.topic.separator.enabled" href="#mirror_source_replication.policy.internal.topic.separator.enabled">replication.policy.internal.topic.separator.enabled</a></h4>
```

Comment:
Was this a forgotten documentation chunk that we forgot to add until now?

Reply:
No, https://issues.apache.org/jira/browse/KAFKA-15102 fixes for 3.5.2 and many other versions.